pkg/cli/tail_test.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !race
+// +build !race
+
```

Comment:
This is generally not a good idea, sine it means there's a data race. What is the race error you're getting?

Reply:
Changed to `-f` and `-follow`. and solved the race condition problem.